README.md

```diff
@@ -250,7 +250,8 @@ var audioVideoConfig = AudioVideoConfiguration(
     audioMode: .mono48k,
     audioDeviceCapabilities: .outputOnly,
     callKitEnabled: true,
-    enableAudioRedundancy: false)
+    enableAudioRedundancy: false, 
+    reconnectTimeoutMs: 180 * 1000)
```

Comment:
Do we need a simple guide on how to configure reconnection timeouts?

Reply:
It's a small change, easy to use. Don't think it's needed, I already added it to README.